helm/aws-vpc-operator/values.yaml

```diff
@@ -0,0 +1,26 @@
+name: aws-vpc-operator
+serviceType: managed
+
+project:
+  branch: "[[ .Branch ]]"
+  commit: "[[ .SHA ]]"
+
+image:
+  registry: quay.io
+  name: giantswarm/aws-vpc-operator
+  tag: v0.1.0
+  pullPolicy: IfNotPresent
+
+pod:
+  user:
+    id: "65532"
+  group:
+    id: "65532"
+  credentials:
+    dir: /home/.aws
+    filename: credentials
```

Comment:
Does this need to be configurable? If I change it wont the app break because it can't find the creds?

Reply:
I added it here for the sake of not copy-pasting path/file name across volumes, volume mount and env var.

It is wired like this (wherever we use those values), so I believe it should work even if changed as `AWS_SHARED_CREDENTIALS_FILE` will get updated as well:
```
        volumeMounts:
        - mountPath: {{ .Values.pod.credentials.dir }}
          name: {{ .Values.pod.credentials.filename }}
        env:
        - name: AWS_SHARED_CREDENTIALS_FILE
          value: {{ .Values.pod.credentials.dir }}/{{ .Values.pod.credentials.filename }}
      volumes:
      - name: {{ .Values.pod.credentials.filename }}
        secret:
          secretName: {{ include "resource.default.name" . }}-aws-credentials

```

Reply:
> I added it here for the sake of not copy-pasting path/file name across volumes, volume mount and env var

I remember us having problems with apps due to inconsistent manifests where stuff gets updated in one place, but not in another (it was some of CAPI apps), so parts of different manifests get out of sync and the app breaks. Hence trying to have a single source of truth for the credentials path here, and wiring it where needed.